docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md

```diff
@@ -1,100 +1,98 @@
 
-Table 2 lists the limits and describes the differences in the portion of the item body that the each of the Outlook applies a regular expression. Some of these limits depend on the type of device and item body, if the regular expression is applied on the item body.
+### Limits on the size of the item body evaluated
 
-**Table 2. Limits on the size of the item body evaluated**
+The following table lists the limits and describes the differences in the portion of the item body to which each Outlook client applies a regular expression. Some of these limits depend on the type of device and item body, if the regular expression is applied on the item body.
 
-||Outlook rich client|Outlook on mobile devices|Outlook on the web|
+||Outlook on Windows and on Mac|Outlook on mobile devices|Outlook on the web|
 |:-----|:-----|:-----|:-----|
 |**Form factor**|Any supported device.|Android smartphones, iPad, or iPhone.|Any supported device other than Android smartphones, iPad, and iPhone.|
 |**Plain text item body**|Applies the regex on the first 1 MB of the data of the body, but not on the rest of the body over that limit.|Activates the add-in only if the body < 16,000 characters.|Activates the add-in only if the body < 500,000 characters.|
 |**HTML item body**|Applies the regex on the first 512 KB of the data of the body, but not on the rest of the body over that limit. (The actual number of characters depends on the encoding which can range from 1 to 4 bytes per character.)|Applies the regex on the first 64,000 characters (including HTML tag characters), but not on the rest of the body over that limit.|Activates the add-in only if the body < 500,000 characters.|
 
-Table 3 lists the limits and describes the differences in the matches that each Outlook client returns after evaluating a regular expression. The support is independent of any specific type of device, but may depend on the type of item body, if the regular expression is applied on the item body.
+### Limits on the matches returned
 
-**Table 3. Limits on the matches returned**
+Table 3 lists the limits and describes the differences in the matches that each Outlook client returns after evaluating a regular expression. The support is independent of any specific type of device, but may depend on the type of item body, if the regular expression is applied on the item body.
```

Comment:
For consistency with your other updates...

```suggestion
The following table lists the limits and describes the differences in the matches that each Outlook client returns after evaluating a regular expression. The support is independent of any specific type of device, but may depend on the type of item body, if the regular expression is applied on the item body.
```

Reply:
Thanks for catching this.